letter-sender/src/main/java/com/christmas/letter/sender/model/ChristmasLetter.java

```diff
@@ -0,0 +1,60 @@
+package com.christmas.letter.sender.model;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+
+public class ChristmasLetter {
+
+    @Email
+    @NotNull(message = "Email is a mandatory field")
+    private String email;
+
+    @NotEmpty(message = "Name is a mandatory field")
+    private String name;
+
+    @NotEmpty(message = "Wishes is a mandatory field")
+    private String wishes;
+
+    @NotEmpty(message = "Location is a mandatory field")
+    private String location;
+
+    public String getEmail() {
```

Comment:
You can use lombok to get rid of all the builerplate code.

Reply:
added lombok and removed the unnecessary methods